frc/jvm/src/test/scala/com/lynbrookrobotics/potassium/frc/CANifierTest.scala

```diff
@@ -0,0 +1,70 @@
+package com.lynbrookrobotics.potassium.frc
+
+import com.ctre.phoenix.CANifier
+import com.lynbrookrobotics.potassium.{ClockMocking, Signal}
+import edu.wpi.first.wpilibj.DriverStation
+import org.scalatest.mockito.MockitoSugar
+import org.scalatest.FunSuite
+import com.lynbrookrobotics.potassium.streams._
+import edu.wpi.first.wpilibj.DriverStation.Alliance
+import org.mockito.Mockito._
+import org.mockito.{ArgumentCaptor, ArgumentMatchers}
+import squants.time.Milliseconds
+
+class CANifierTest extends FunSuite with MockitoSugar{
+
+  test("Test custom color class"){
```

Comment:
Starting the test name with "Test" is redundant. Instead, focus on what the test is evaluating, so for example this could be something like "Custom color class stores RGB values".